topic-offset-service/src/main/java/uk/gov/gchq/palisade/service/topicoffset/service/SimpleTopicOffsetService.java

```diff
@@ -40,10 +39,10 @@
      */
     @Override
     public boolean isOffsetForTopic(final Headers headers) {
-        Header x = headers.lastHeader(StreamMarker.HEADER);
-        boolean isOffset = ((x != null) && (new String(x.value(), Charset.defaultCharset()).equals(StreamMarker.START.toString())));
+        var header = headers.lastHeader(StreamMarker.HEADER);
+        boolean isOffset = ((header != null) && (new String(header.value(), Charset.defaultCharset()).equals(StreamMarker.START.toString())));
         if (isOffset) {
-            LOGGER.info("Found topic offset: {}", headers);
+            LOGGER.debug("Found topic offset: {}", headers);
         }
```

Comment:
Keep at info, we only see one of these messages per request (like palisade, user service)